app/assets/stylesheets/availabilities.scss

```diff
@@ -83,3 +123,193 @@
   top: -2px;
   float: left;
 }
+
+.availabilities-all-cell {
+  border: 1px solid #000 !important;
+}
+
+.availability-list {
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+li.text-overflow-ellipsis {
+  list-style-position: inside;
+}
+
+.body {
+  text-rendering: geometricPrecision;
+}
+
+html, body {
+  height: 100%;
+}
+
+.places-container {
+  height: calc(100% - 71px);
+}
+
+.edit-places-form {
+  width: 100%;
+  height: 100%;
+}
+
+.edit-places-nav {
+  height: 100%;
+  border-right: 1px solid #e0e0e0;
```

Comment:
Do abstract the `#e0e0e0` into a SCSS variable

Reply:
Done